### ISSUE TITLE
Silencing "Optimizing fused batch norm node" stdout during model loading

### ISSUE THREAD

**Issue by pawarren** [nwojke/deep_sort#99]
I can't figure out how to totally silence the loading stdout when I load the Tensorflow-based feature extractor.

Here is what I can't get rid of:

```
Optimizing fused batch norm node name: "net/conv4_3/1/conv4_3/1/bn/FusedBatchNorm"
op: "FusedBatchNorm"
input: "net/conv4_3/1/Conv2D"
input: "net/conv4_3/1/conv4_3/1/bn/Const"
input: "net/conv4_3/1/conv4_3/1/bn/beta"
input: "net/conv4_3/1/conv4_3/1/bn/moving_mean"
input: "net/conv4_3/1/conv4_3/1/bn/moving_variance"
device: "/job:localhost/replica:0/task:0/device:GPU:0"
attr {
  key: "T"
  value {
    type: DT_FLOAT
  }
}
attr {
  key: "data_format"
  value {
    s: "NHWC"
  }
}
attr {
  key: "epsilon"
  value {
    f: 0.001
  }
}
attr {
  key: "is_training"
  value {
    b: false
  }
}
```

I've added the following verbosity variables to the top of the `generate_detections.py` file. I've tried many combination of settings but nothing has worked.

```
import os
os.environ['TF_CPP_MIN_VLOG_LEVEL'] = '0'
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
from tensorflow import logging
logging.set_verbosity(logging.FATAL)
```

What else can I try?
